openquake/hazardlib/gsim/base.py

```diff
@@ -367,8 +367,6 @@ def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
         if self.compute.__annotations__.get("ctx") is numpy.recarray:
             cmaker = ContextMaker('*', [self], {'imtls': {imt.string: [0]},
                                                 'mags': ['%.2f' % rup.mag]})
-            if not isinstance(ctx, numpy.ndarray):
-                ctx = cmaker.recarray([ctx])
         assert self.compute.__annotations__.get("ctx") is numpy.recarray
         cmaker = simple_cmaker([self], [imt.string])
```

Comment:
It seems a merge conflict was manage incorrectly here. I guess lines 367,368,369 should disappear and line 371 become
```python
 cmaker = simple_cmaker([self], [imt.string], mags=['%.2f' % rup.mag]})
```